classes/class-ledyer-main.php

```diff
@@ -51,7 +51,7 @@ class Ledyer_Checkout_For_WooCommerce {
 	 */
 	public $checkout;
 
-	const VERSION = '1.7.8';
+	const VERSION = '1.8.0';
```

Comment:
not sure if versioning is handled by CI

Reply:
Good point I've removed the CI for now as it's not helping